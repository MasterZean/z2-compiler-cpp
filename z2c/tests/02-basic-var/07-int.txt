val d1 = 0s;
val d2 = -1s
val d3 = 1s;
val d4 = -100s
val d5 = 100s;
val d6 = -127s
val d7 = 127s;
val d8 = -128s
val d9 = 128s;
val d10 = -32767s
val d11 = 32767s;
val d12 = -32768s
val d13 = 32768s;
val d14 = -2147483647s
val d15 = 2147483647s;
val d16 = -2147483648s