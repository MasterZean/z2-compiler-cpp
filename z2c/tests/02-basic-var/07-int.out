int32 d1 = 0;
int32 d2 = -1;
int32 d3 = 1;
int32 d4 = -100;
int32 d5 = 100;
int32 d6 = -127;
int32 d7 = 127;
int32 d8 = -128;
int32 d9 = 128;
int32 d10 = -32767;
int32 d11 = 32767;
int32 d12 = -32768;
int32 d13 = 32768;
int32 d14 = -2147483647;
int32 d15 = 2147483647;
int32 d16 = (int32)-2147483648ll;
